pkg/match/README.md

```diff
@@ -0,0 +1,19 @@
+### Bench
+
+For the fun of it, I benched my brute-force version along with my second post-epiphany version. No surprise.
```

Comment:
![curiosity](https://user-images.githubusercontent.com/11455210/110298477-ed9a6b00-7ff4-11eb-9b33-37b043e3b3b5.gif)

Reply:
(i.e. tell me what to look and how to understand this)